addons/l10n_ar/demo/account_supplier_invoice_demo.xml

```diff
@@ -25,6 +25,7 @@
     <!-- Invoice from Foreign with vat 21, 27 and 10,5 -->
     <record id="demo_sup_invoice_2" model="account.move">
         <field name="partner_id" ref="res_partner_foreign"/>
+        <field name="fiscal_position_id" model="account.fiscal.position" eval="obj().search([('company_id', '=', obj().env.ref('l10n_ar.company_ri').id), ('name', '=', 'Compras / Ventas al exterior')], limit=1).id"/>
```

Comment:
I know it has already been used, but I wonder if it would not be better to do it with the onchange.  Comparing with name is a bit weird.  (I know template and instance)  I don't know if you tested that.  

Also in this file, there still is a commented section about the manual taxes.  Have not we solved that already?  I would like to avoid such comments in the code.